DqtApi/src/DqtApi/Services/TrnGenerationApi/ITrnGenerationApiClient.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DqtApi.Services.TrnGenerationApi
+{
+    using System.Threading.Tasks;
```

Comment:
Nit: `using`s outside namespace

Reply:
Fixed and sorted my Visuals Studio settings!